pycalc11/interface.py

```diff
@@ -82,7 +82,7 @@ def require_rerun(self):
     def __init__(self, station_names=None, station_coords=None, source_coords=None,
             start_time=None, duration_min=None,
             base_mode='geocenter', uvw_mode='exact', dry_atm=True, wet_atm=True,
-            calc_file=None,
+            calc_file=None,d_interval=1,
```

Comment:
Let's have this default to 24, since we want the default behavior to be equivalent to difxcalc11.
Also need a space between the comma and keyword (`, d_interval`)